readme-slack.md

```diff
@@ -1,35 +1,39 @@
 # BotMan with Slack
 
-**Note:**
-
-You have three possibilities to set up Slack to connect with BotMan.
+**Note:** You have three possibilities to set up Slack to connect with BotMan:
 
 #### Use a bot with the Slack Realtime API
 
 **Pros:** 
    * Your bot user will be a real bot and be able to join channels / talk to in direct messages
    * Very easy to set up
 
-> **Note:** Until now, you can not yet use  [interactive message buttons](https://api.slack.com/docs/message-buttons) with BotMan and Slack Realtime API.
+> **Note:** As of now, you can not yet use [interactive message buttons](https://api.slack.com/docs/message-buttons) with BotMan and Slack Realtime API.
```

Comment:
I think what you're saying is that you can't use interactive buttons right now correct? If so then I think this reads a little better.